project-base/src/Shopsys/ShopBundle/DataFixtures/ProductDataFixtureReferenceInjector.php

```diff
@@ -3,188 +3,176 @@
     {
         return [
-            'high' => $persistentReferenceFacade->getReference(VatDataFixture::VAT_HIGH),
-            'low' => $persistentReferenceFacade->getReference(VatDataFixture::VAT_LOW),
-            'second_low' => $persistentReferenceFacade->getReference(VatDataFixture::VAT_SECOND_LOW),
-            'zero' => $persistentReferenceFacade->getReference(VatDataFixture::VAT_ZERO),
+            'high' => $this->persistentReferenceFacade->getReference(VatDataFixture::VAT_HIGH),
+            'low' => $this->persistentReferenceFacade->getReference(VatDataFixture::VAT_LOW),
+            'second_low' => $this->persistentReferenceFacade->getReference(VatDataFixture::VAT_SECOND_LOW),
+            'zero' => $this->persistentReferenceFacade->getReference(VatDataFixture::VAT_ZERO),
         ];
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\DataFixture\PersistentReferenceFacade $persistentReferenceFacade
      * @return string[]
      */
-    private function getAvailabilityReferences(PersistentReferenceFacade $persistentReferenceFacade)
+    public function getAvailabilityReferences()
     {
         return [
-            'in-stock' => $persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_IN_STOCK),
-            'out-of-stock' => $persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_OUT_OF_STOCK),
-            'on-request' => $persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_ON_REQUEST),
+            'in-stock' => $this->persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_IN_STOCK),
+            'out-of-stock' => $this->persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_OUT_OF_STOCK),
+            'on-request' => $this->persistentReferenceFacade->getReference(AvailabilityDataFixture::AVAILABILITY_ON_REQUEST),
         ];
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\DataFixture\PersistentReferenceFacade $persistentReferenceFacade
      * @return string[]
      */
-    private function getCategoryReferences(PersistentReferenceFacade $persistentReferenceFacade)
+    public function getCategoryReferences()
     {
         return [
-            'electronics' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_ELECTRONICS),
-            'tv' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_TV),
-            'photo' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_PHOTO),
-            'printers' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_PRINTERS),
-            'pc' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_PC),
-            'phones' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_PHONES),
-            'coffee' => $persistentReferenceFacade->getReference(CategoryDataFixture::CATEGORY_COFFEE),
```

Comment:
looks like some of those were removed

Reply:
No, this class is now used diferently, it is more Provider then Injector.